### ISSUE TITLE
On "fatal errors" stop serving files

### ISSUE THREAD

**Issue by TechplexEngineer** [qooxdoo/qooxdoo-compiler#608]
When developing, if I make a syntax error and am using `qx serve` there is no indication in the browser that the compilation failed.

Would it be possible to instead print an error in the browser this way if the developer is not closely monitoring the console where the `qx serve` is running gets an earlier indication that their compile failed?

**Comment by johnspackman** [comment]
i completely agree; I would also like it if there are just warnings, those warnings are picked up and output into the console during app startup
